### PR TITLE
delete hightlight and downplay doc

### PR DESCRIPTION
**Pull request by gitforhlp**
fix [issues#185](https://github.com/apache/echarts-doc/issues/185)

### CODE REVIEW

File: en/api/action.md [hunk 1]
```diff
@@ -36,54 +36,6 @@ Chart actions supported by ECharts are triggered through [dispatchAction](~echar
 
 **Attention: ** The `?:` note in the code shows that this attribute is optional. *EVENT:* stands for the event that triggers action.
 
-## highlight(Action)
```
Comment (plainheart) [inline en/api/action.md:39]: We shouldn't remove them as they are still working for other series or components.

The v5 upgrade guide indicates that the `highlight` action & `downplay` action in the `sunburst` series are deprecated but not in all series.

Perhaps we should add new actions for `sunburstHighlight` & `sunburstUnhighlight`.

**Review by plainheart — CHANGES_REQUESTED** [review]

**Comment by Ovilia** [comment]
@pissang Please help review this PR.

**Comment by pissang** [comment]
As @plainheart said, this change is only scoped to the sunburst series, it should be in mentioned in the note

**Comment by pissang** [comment]
I was confused because `highlight` is new introduced since 5.0 and it should not be deprecated. So I just did some search in the code and it seems the warning log is wrong in https://github.com/apache/echarts/blob/master/src/chart/sunburst/sunburstAction.ts#L90

It should be.
```ts
deprecateReplaceLog('sunburstHighlight', 'highlight');
```

So we should change this warning log instead of changing the doc

**Comment by gitforhlp** [comment]
> I was confused because `highlight` is new introduced since 5.0 and it should not be deprecated. So I just did some search in the code and it seems the warning log is wrong in https://github.com/apache/echarts/blob/master/src/chart/sunburst/sunburstAction.ts#L90
> 
> It should be.
> 
> ```ts
> deprecateReplaceLog('sunburstHighlight', 'highlight');
> ```
> 
> So we should change this warning log instead of changing the doc

It is stated in the V5 document that this is discarded in the sunrise chart, and it is not a code problem
[](https://echarts.apache.org/handbook/zh/basics/release-note/v5-upgrade-guide#%E4%B8%8D%E5%86%8D%E6%8E%A8%E8%8D%90%E4%BD%BF%E7%94%A8%E7%9A%84-api)

please review again

**Comment by gitforhlp** [comment]
> I was confused because `highlight` is new introduced since 5.0 and it should not be deprecated. So I just did some search in the code and it seems the warning log is wrong in https://github.com/apache/echarts/blob/master/src/chart/sunburst/sunburstAction.ts#L90
> It is stated in the V5 document that this is discarded in the sunrise chart, and it is not a code problem
> It should be.
> 
> ```ts
> deprecateReplaceLog('sunburstHighlight', 'highlight');
> ```
> 
> So we should change this warning log instead of changing the doc

It is stated in the V5 document that this is discarded in the sunrise chart, and it is not a code problem,please check again
[https://echarts.apache.org/handbook/zh/basics/release-note/v5-upgrade-guide/#%E4%B8%8D%E5%86%8D%E6%8E%A8%E8%8D%90%E4%BD%BF%E7%94%A8%E7%9A%84-api](url)

**Comment by gitforhlp** [comment]
> I was confused because `highlight` is new introduced since 5.0 and it should not be deprecated. So I just did some search in the code and it seems the warning log is wrong in https://github.com/apache/echarts/blob/master/src/chart/sunburst/sunburstAction.ts#L90
> 
> It should be.
> 
> ```ts
> deprecateReplaceLog('sunburstHighlight', 'highlight');
> ```
> 
> So we should change this warning log instead of changing the doc
It is stated in the V5 document that this is discarded in the sunrise chart, please check again
[https://echarts.apache.org/handbook/zh/basics/release-note/v5-upgrade-guide/#%E4%B8%8D%E5%86%8D%E6%8E%A8%E8%8D%90%E4%BD%BF%E7%94%A8%E7%9A%84-api](url)

**Comment by gitforhlp** [comment]
It is stated in the V5 upgrade document that this is discarded in the sunrise chart,please check again
https://echarts.apache.org/handbook/zh/basics/release-note/v5-upgrade-guide#%E4%B8%8D%E5%86%8D%E6%8E%A8%E8%8D%90%E4%BD%BF%E7%94%A8%E7%9A%84-api

旭日图（sunburst）：
动作类型 highlight 已被弃用，请使用 sunburstHighlight 代替。
动作类型 downplay 已被弃用，请使用 sunburstUnhighlight 代替。
选项 series.downplay 已被弃用，请使用 series.blur 代替。
选项 series.highlightPolicy 已不适用，请使用 series.emphasis.focus 代替。

**Comment by Ovilia** [comment]
apache/echarts#17587 is following the right track. I think there should be no document changing required so I'm going to close this PR. Thanks for the help!

**Pull request closed without merging**